composer.json

```diff
@@ -8,31 +8,37 @@
         {
             "name": "Jérôme Gamez",
             "email": "jerome@gamez.name"
+        },{
+            "name": "Gary PEGEOT",
+            "email": "garypegeot@gmail.com"
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
-        "php-http/plugins": "^1.0",
-        "guzzlehttp/psr7": "^1.2"
+        "php": "^7.1",
+        "guzzlehttp/psr7": "^1.4",
+        "php-http/client-common": "^2.0",
+        "psr/log": "^1.0",
+        "symfony/filesystem": "^3.4|^4.0",
+        "symfony/options-resolver": "^3.4|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.2",
-        "mikey179/vfsStream": "^1.6"
+        "symfony/phpunit-bridge": "*",
```

Comment:
this likely needs to be locket to some recent version because old versions (in the lowest-build) try downloading phpunit versions from github that are no longer provided.